ports/nrf/boards/arduino_nano_33_ble/modules/hts221.py

```diff
@@ -0,0 +1,76 @@
+"""
+The MIT License (MIT)
+
+Copyright (c) 2013-2022 Ibrahim Abdelkader <iabdalkader@openmv.io>
+Copyright (c) 2013-2022 Kwabena W. Agyeman <kwagyeman@openmv.io>
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+
+HTS221 driver driver for MicroPython.
+Original source: https://github.com/ControlEverythingCommunity/HTS221/blob/master/Python/HTS221.py
+"""
+
+import struct
+
+
+class HTS221:
+    def __init__(self, i2c, data_rate=1, dev_addr=0x5F):
```

Comment:
maybe rename `dev_addr` to `address`, to at least match LSM6DSOX

Reply:
They still have different args, and example the LSM9 accepts more than one address (`address_gyro` and `address_magnet`), so not sure what's the point of renaming that particular arg, but okay.

Reply:
It's good to try and be consistent, so you don't have to remember that this driver uses `dev_addr` and that driver uses `address` and this other one uses `addr`.  Also it saves on code size by reusing the same qstr.